cmd/lncli/cmd_pay.go

```diff
@@ -147,12 +147,13 @@ var sendPaymentCommand = cli.Command{
 	If payment isn't manually specified, then only a payment request needs
 	to be passed using the --pay_req argument.
 
-	If the payment *is* manually specified, then all four alternative
+	If the payment *is* manually specified, then all five alternative
 	arguments need to be specified in order to complete the payment:
 	    * --dest=N
 	    * --amt=A
 	    * --final_cltv_delta=T
 	    * --payment_hash=H
+	    * --pay_addr=H
```

Comment:
Should we support using positional arguments for this as well? Just for consistency...

Reply:
Yes! Changed.